TwipsterApp/Validators/ChangedPasswordValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using TwipsterApp.Interfaces;
+
+namespace TwipsterApp.Validators
+{
+    public class ChangedPasswordValidator : IValidator
+    {
```

Comment:
Create ctor here and use argument in ctor. In void Validate() function should be only logic to validate needed instance.